modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -40,6 +40,14 @@ Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft A
 | 25,000
 | 25,000
 
+| Number of routes and backends per Ingress Controller
+| 2,000 per router
+| 2,000 per router
+
+| Number of secrets and configmaps
```

Comment:
configmaps should be config maps per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

Also, use back ends instead of backend per IBM Style guide